src-java/kilda-model/src/main/java/org/openkilda/model/cookie/CookieBase.java

```diff
@@ -155,6 +155,7 @@ protected static String formatIllegalTypeError(CookieType illegalType, Set<Cooki
         APPLICATION_MIRROR_FLOW(0x00A),
         EXCLUSION_FLOW(0x0B),
         SERVER_42_INGRESS(0x00C),
+        MIRROR_FLOW(0x00D),
```

Comment:
Do we really need new cookie type? Can we use existing SERVICE_OR_FLOW_SEGMENT and add new "mirror" flag in it?

Reply:
Yes, we can use existing SERVICE_OR_FLOW_SEGMENT and add new "mirror" flag in it. It was done.